### PR TITLE
Add chatwoot template

### PR DESCRIPTION
**Pull request by odraude7**
Chatwoot template
- Rails
- Sidekiq
- Postgres
- Redis

### CODE REVIEW

**Comment by andrasbacsai** [comment]
Thank you for the PR! I will test it!

**Comment by AlexBSoft** [comment]
While using Telegram inbox - there an issue - **images sent by users are not shown in chat.** In dev tools rails active storage files returns error 500
image

Also i am not enable to reconfigure storage in coolify. 
Env variables `ACTIVE_STORAGE_SERVICE`, `STORAGE_BUCKET_NAME`, `STORAGE_ACCESS_KEY_ID`, `STORAGE_REGION`, `STORAGE_ENDPOINT` (as in [docs](https://www.chatwoot.com/docs/self-hosted/deployment/storage/supported-providers#using-amazon-s3-compatible-service)) when added to Environment Variables in coolify seems to be not consumed and Chatwoot still tries to use active storage.

File: templates/compose/chatwoot.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      - INSTALLATION_ENV=docker
+      - MAILER_SENDER_EMAIL=${CHATWOOT_MAILER_SENDER_EMAIL}
+      - SMTP_ADDRESS=${CHATWOOT_SMTP_ADDRESS}
+      - SMTP_AUTHENTICATION=${CHATWOOT_SMTP_AUTHENTICATION}
+      - SMTP_DOMAIN=${CHATWOOT_SMTP_DOMAIN}
+      - SMTP_ENABLE_STARTTLS_AUTO=${CHATWOOT_SMTP_ENABLE_STARTTLS_AUTO}
+      - SMTP_PORT=${CHATWOOT_SMTP_PORT}
+      - SMTP_USERNAME=${CHATWOOT_SMTP_USERNAME}
+      - SMTP_PASSWORD=${CHATWOOT_SMTP_PASSWORD}
+      - ACTIVE_STORAGE_SERVICE=local
```
Comment (AlexBSoft) [inline templates/compose/chatwoot.yaml:40]: We need option to change active storage type to use S3 storage

**Comment by AlexBSoft** [comment]
I have figured out how to fix the problem, simply replace existing ACTIVE_STORAGE_SERVICE line with this lines:

```
      - ACTIVE_STORAGE_SERVICE=${ACTIVE_STORAGE_SERVICE:-local}
      - STORAGE_BUCKET_NAME=${STORAGE_BUCKET_NAME}
      - STORAGE_ACCESS_KEY_ID=${STORAGE_ACCESS_KEY_ID}
      - STORAGE_SECRET_ACCESS_KEY=${STORAGE_SECRET_ACCESS_KEY}
      - STORAGE_REGION=${STORAGE_REGION}
      - STORAGE_ENDPOINT=${STORAGE_ENDPOINT}
      - STORAGE_FORCE_PATH_STYLE=${STORAGE_FORCE_PATH_STYLE:-false}
```

And then add this variables in coolify

My template with enabled custom S3: https://github.com/fullhouseit/chatwoot-coolify-template/blob/main/docker-compose.yaml

**Comment by Neyunse** [comment]
I wanted to use a custom subdomain for the help center but I am not able to add it, any ideas? it seems that it is not configured correctly.

**Pull request merged**